app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+
+const app = express();
+
+app.listen(3000, () => {
+  console.log("Our server is READY!");
+});
+
+app.use(express.static(__dirname + "/public"));
+
+app.get("/", (request, response, next) => {
+  response.sendfile(__dirname + "/views/home-page.html");
+});
+
+app.get("/about-page", (request, response, next) => {
```

Comment:
normally, in the URL you would just include ``/about `` because it makes it easier for the user to navigate to the page